src/main/java/be/acara/frontend/exception/ExceptionController.java

```diff
@@ -1,32 +1,48 @@
 package be.acara.frontend.exception;
 
 import feign.FeignException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.access.AccessDeniedException;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.GetMapping;
 
 @ControllerAdvice
 @Controller
+@Slf4j
 public class ExceptionController {
     
+    private static final String UNAUTHORIZED = "error/unauthorized";
+    
     @ExceptionHandler(FeignException.class)
     public String handleFeignClientExceptions(FeignException fe) {
         switch (fe.status()) {
             case 401:
-            case 403: return "error/unauthorized";
+            case 403: return UNAUTHORIZED;
```

Comment:
For consistency's sake, it might be a good idea to extract these other paths to constant variables as well